images/master/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM google/debian:wheezy
+
+ADD kube-apiserver /kube-apiserver
+ADD kube-controller-manager /kube-controller-manager
+ADD kube-scheduler /kube-scheduler
```

Comment:
I don't think hyperkube is there yet.  That's the eventual trajectory.  In the meantime, building an distributing a single image is easier.